src/commands/update.cr

```diff
@@ -14,6 +14,7 @@ module Shards
             Shards.logger.info "Installing #{package.name} (#{package.version})"
             package.install
           end
+          package.install_executables
```

Comment:
Hum, executables haven't been rebuilt if the dependency was already installed, why bother reinstalling it? Or maybe a loose version (reinstall if missing?)